### PR TITLE
Fix API report newlines

### PR DESCRIPTION
**Pull request by ecraig12345**
#### Pull request checklist

- [x] Include a change request file using `$ yarn change`

#### Description of changes

Normalize API report file newlines and remove trailing spaces. While this may cause inconvenience in merging in the short term, longer-term it will make API report diffs much more useful.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/11104)

### CODE REVIEW

**Comment by octogonz** [comment]
Why not implement this feature in API Extractor?  The design and partial work was already laid out in: https://github.com/microsoft/rushstack/issues/1366#issuecomment-537740747

**Comment by ecraig12345** [comment]
Because I saw some previous issues (would have to dig them up again) where you were against normalizing the newlines based on the operating system standard, which is what this option does. The trailing space thing would be nice to fix in api-extractor though.

**Comment by octogonz** [comment]
Hmm... if you come across them again, lemme know so we clear up any confusion.  (Searching for `crlf` turns up https://github.com/microsoft/rushstack/issues/1467 and https://github.com/microsoft/rushstack/issues/1366 which both are advocating to support `\n`.)

In any case, how about fixing it upstream? 😊

File: packages/lists/package.json [hunk 1]
```diff
@@ -20,7 +20,6 @@
     "code-style": "just-scripts code-style",
     "start": "just-scripts dev",
     "start-test": "just-scripts jest-watch",
-    "update-api": "just-scripts update-api",
```
Comment (KevinTCoughlin) [inline packages/lists/package.json]: This script and associated output file will have to get reintroduced once @soettl introduces downstream List improvements. The blocker _was_ Fabric 7 upgrade, but we've recently migrated and should be unblocked. I'd keep it.

File: packages/test-utilities/etc/test-utilities.api.md [hunk 2]
```diff
@@ -1,20 +0,0 @@
...
-
-import * as React from 'react';
-import { ReactWrapper } from 'enzyme';
-import * as renderer from 'react-test-renderer';
-
-// @public
-export function safeCreate<TProps>(content: React.ReactElement<TProps>, callback: (wrapper: renderer.ReactTestRenderer) => void): void;
-
-// @public
-export function safeMount<TComponent extends React.Component, TProps = TComponent['props'], TState = TComponent['state']>(content: React.ReactElement<TProps>, callback: (wrapper: ReactWrapper<TProps, TState, TComponent>) => void): void;
```
Comment (KevinTCoughlin) [inline packages/test-utilities/etc/test-utilities.api.md]: Is this removal intentional? This package `@uifabric/test-utilities` has users: https://www.npmjs.com/package/@uifabric/test-utilities. I recommend keeping the file and associated script.

File: packages/lists/package.json (hunk 1 above)
Comment (ecraig12345) [reply packages/lists/package.json]: The task isn't actually hooked up to do anything at the moment (api-extractor doesn't run as part of lists build), so I thought it would make more sense to get rid of it for now and re-introduce properly if/when it's actually needed.

Comment (ecraig12345) [reply packages/lists/package.json]: Basically make it clear that api-extractor is **not** running for this project until Sebastian is ready to set it up, rather than having some elements (the api.md and script) which make it appear to be running when it's actually not.

File: packages/test-utilities/etc/test-utilities.api.md (hunk 2 above)
Comment (ecraig12345) [reply packages/test-utilities/etc/test-utilities.api.md]: The API file wasn't getting generated as part of this package's build (so the rollup probably hasn't been generated or published for awhile either). I'm not sure whether that was intentional, or whether this package really needs an API review file and rollup.

Comment (dzearing) [reply packages/test-utilities/etc/test-utilities.api.md]: Any update here? the PR is getting stale :)

Comment (ecraig12345) [reply packages/test-utilities/etc/test-utilities.api.md]: I ended up fixing it in api-extractor, so I'll open another PR to pick up that fix instead.
